### PR TITLE
Add Actian Zen Charts

### PR DESCRIPTION
**Pull request by iazrai-actian**

### CODE REVIEW

File: applications/actian-zen/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Copyright (c) 2020 Actian Corporation
+```
+# Actian Zen 
+Actian Zen Edge Data Management is a single, secure, modular, and scalable solution that embeds in or bundles with applications for remote/branch, mobile, and IoT data processing and analytics support. Zen provides SQL and NoSQL access for any type of data from within most popular programming languages running on Windows (IoT Core, Desktop, Server), Linux (embedded and standard), MacOS running on Intel platforms and running on iOS/Android for mobile.
+
+For more information on Actian Zen go to https://www.actian.com/data-management/zen-embedded-database/
+
+## Install
+Helm charts are provided in the 'helm-charts' directory to assist with the deployment of the Zen database and sample application in an OpenNESS environment.
+To obtain the Zen database container and the sample application container please go to https://github.com/ActianCorp/ActianZen-OpenNESS .
```
Comment (amr-mokhtar) [inline applications/actian-zen/README.md:12]: This repo is empty at the moment. I assume that you plan to push the containers in the near future?

File: applications/actian-zen/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+```text
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2020 Actian Corporation
+```
+# Actian Zen 
+Actian Zen Edge Data Management is a single, secure, modular, and scalable solution that embeds in or bundles with applications for remote/branch, mobile, and IoT data processing and analytics support. Zen provides SQL and NoSQL access for any type of data from within most popular programming languages running on Windows (IoT Core, Desktop, Server), Linux (embedded and standard), MacOS running on Intel platforms and running on iOS/Android for mobile.
+
+For more information on Actian Zen go to https://www.actian.com/data-management/zen-embedded-database/
+
+## Install
```
Comment (amr-mokhtar) [inline applications/actian-zen/README.md:10]: It is always good to provide some installation steps here for the users to quickly get started. See [Smart City App Readme](https://github.com/open-ness/edgeapps/blob/master/applications/smart-city-app/README.md)

File: applications/actian-zen/helm-charts/actianzen/values.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020 Actian Corporation
+
+replicaCount: 1
+
+image:
+  name: actianzen
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
```
Comment (amr-mokhtar) [inline applications/actian-zen/helm-charts/actianzen/values.yaml:9]: Comment is unclear. Is the user need to take care of that? Can you clarify how the user can identify the correct tag?

File: applications/actian-zen/helm-charts/actianzen/values.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
```
Comment (amr-mokhtar) [inline applications/actian-zen/helm-charts/actianzen/values.yaml:15]: Just a cosmetic comment: There is no need for the empty new lines between the yaml statements

File: applications/actian-zen/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+Helm charts are provided in the 'helm-charts' directory to assist with the deployment of the Zen database and sample application in an OpenNESS environment.
```
Comment (amr-mokhtar) [inline applications/actian-zen/README.md:11]: Can you elaborate more on the purposes of the two helm charts provided: `actianzen` and `zensample` ?

**Review by amr-mokhtar — COMMENTED** [review]
The package helm chart (applications/actian-zen/helm-charts/zensample/charts/actianzen-0.1.0.tgz) is not really needed as we already have the chart source files.
Also, it is generally advised to avoid uploading binary packages to avoid bloating the repo.

File: applications/actian-zen/helm-charts/actianzen/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+apiVersion: v2
+name: actianzen
+version: 0.1.0
+appVersion: 14.11.23
+description: Helm chart for deploying Actian Zen Enterprise Server
+home: https://www.actian.com/data-management/zen-embedded-database/
+icon: https://www.actian.com/wp-content/uploads/2020/01/Actian-Logo-2020.png
+sources:
+   - https://github.com/ActianCorp/TBD
+kewords:
```
Comment (pburiez) [inline applications/actian-zen/helm-charts/actianzen/Chart.yaml:13]: Typo: "keywords" expected here

File: applications/actian-zen/README.md (hunk 1 above)
Comment (iazrai-actian) [reply applications/actian-zen/README.md:12]: Yes, they are up now.

File: applications/actian-zen/README.md (hunk 2 above)
Comment (iazrai-actian) [reply applications/actian-zen/README.md:10]: Installation steps added in latest commit.

File: applications/actian-zen/helm-charts/actianzen/values.yaml (hunk 3 above)
Comment (iazrai-actian) [reply applications/actian-zen/helm-charts/actianzen/values.yaml:9]: Comment removed as it was confusing. Currently we are providing archives of the containers with only one tag so elaborating on this isn't necessary at the moment.

File: applications/actian-zen/helm-charts/actianzen/values.yaml (hunk 4 above)
Comment (iazrai-actian) [reply applications/actian-zen/helm-charts/actianzen/values.yaml:15]: Fixed.

File: applications/actian-zen/README.md (hunk 5 above)
Comment (iazrai-actian) [reply applications/actian-zen/README.md:11]: Additional info provided in latest commit.

File: applications/actian-zen/helm-charts/actianzen/Chart.yaml (hunk 6 above)
Comment (iazrai-actian) [reply applications/actian-zen/helm-charts/actianzen/Chart.yaml:13]: Fixed.

File: applications/actian-zen/README.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+# Application Overview
+Two helm charts are provided in the 'helm-charts' directory to assist with Zen deployments in an OpenNESS environment: an Actian Zen database chart and a sample application chart. The 'actianzen' chart will deploy the Actian Zen database as a service in the OpenNESS Kubernetes environment with a 30 day trial license. The 'zensample' chart deploys a simple ODBC web application that communicates with a Zen database. The Zen sample chart has the Zen database chart configured as a dependency which will deploy both the Zen database and the sample application containers when installed.
+
+## Deploying the Zen Database and Sample Application
+
+### Loading Docker Images
+A prerequisite for deploying the Zen database and sample application is to "pre-pull" the docker container images onto the edge nodes. To obtain the Zen database container and the sample application container please go to https://github.com/ActianCorp/ActianZen-OpenNESS to download the archived images.
+
+Once downloaded and copied to the edge nodes, the Zen database and sample application container images can be loaded using the following commands:
```
[the remark this answers is not in the document]
Comment (iazrai-actian) [reply applications/actian-zen/README.md:18]: Will definitely look into utilizing this for the next set of improvements planned for Zen in OpenNESS.

File: applications/actian-zen/helm-charts/actianzen/Chart.yaml [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+apiVersion: v2
+name: actianzen
+version: 0.1.0
+appVersion: 14.11.23
+description: Helm chart for deploying Actian Zen Enterprise Server
+home: https://www.actian.com/data-management/zen-embedded-database/
+icon: https://www.actian.com/wp-content/uploads/2020/01/Actian-Logo-2020.png
+sources:
+   - https://github.com/ActianCorp/TBD
```
[the remark this answers is not in the document]
Comment (iazrai-actian) [reply applications/actian-zen/helm-charts/actianzen/Chart.yaml:12]: Yes it should. Fixed.

File: applications/actian-zen/README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (amr-mokhtar) [reply applications/actian-zen/README.md:18]: That's perfect.

**Pull request merged**